pkg/virt-launcher/virtwrap/util/libvirt_helper_test.go

```diff
@@ -148,22 +148,22 @@ var _ = Describe("LibvirtHelper", func() {
 		for scanner.Scan() {
 			entry := map[string]string{}
 			err := json.Unmarshal(scanner.Bytes(), &entry)
-			Expect(err).To(Not(HaveOccurred()))
+			Expect(err).ToNot(HaveOccurred())
 			//delete(entry, "timestamp")
 			loggedLines = append(loggedLines, entry)
 		}
-		Expect(scanner.Err()).To(Not(HaveOccurred()))
+		Expect(scanner.Err()).ToNot(HaveOccurred())
 
 		expectedLines := []map[string]string{}
 		scanner = bufio.NewScanner(strings.NewReader(formattedLogs))
 		for scanner.Scan() {
 			entry := map[string]string{}
 			err := json.Unmarshal(scanner.Bytes(), &entry)
-			Expect(err).To(Not(HaveOccurred()))
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
A direct approach fits here: `.To(Succeed())`

Reply:
Done